src/api.js

```diff
@@ -49,3 +49,17 @@ export const fetchUser = (username) => {
     return response.data.user;
   });
 };
+
+export const fetchCategories = () => {
+  return gamesApi.get(`/categories`).then((response) => {
+    return response.data.categories;
+  });
+};
+
+export const fetchReviewsCategory = (page, category) => {
+  return gamesApi
+    .get(`/reviews?page=${page}&category=${category}`)
```

Comment:
Please try to refactor this to make use of 'axios params'. See NC Notes: Axios/Axios Params.